cmd/troubleshoot/cli/run_test.go

```diff
@@ -371,3 +371,49 @@ spec:
 	assert.Len(t, sb.Spec.HostCollectors, 1)
 	assert.Len(t, sb.Spec.HostAnalyzers, 1)
 }
+
+func Test_loadSpecsFromURL(t *testing.T) {
+	// Run a webserver to serve the URI spec
+	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte(`
+	apiVersion: troubleshoot.sh/v1beta2
+	kind: SupportBundle
+	metadata:
+	  name: sb-2
+	spec:
+	  collectors:
+	    - logs:
+          name: podlogs/kotsadm
+          selector:
+            - app=kotsadm`))
```

Comment:
Not that it will change the test outcome, but indentation in this spec is not quite right. I think you have tabs instead of spaces

Reply:
thanks @banjoh! I've updated the test.